fakeredis/_server.py

```diff
@@ -1853,7 +1853,7 @@ def ltrim(self, key, start, stop):
                 key.update(new_value)
         return OK
 
-    @command((Key(list),), (Int(),))
```

Comment:
Stupid question: What's the difference?

Reply:
With the old code, the wrapper will immediate raise an exception if the key exists and it is not a list before even trying to call the function. With the new code, the type is checked inside `_list_pop`, *after* some other checks.

Reply:
Thanks, thought as much!